### ISSUE TITLE
training loss fluctuates

### ISSUE THREAD

**Issue by nhatanhphung** [ShiqiYu/libfacedetection.train#29]
Hello, i'm currently training the model using the wider face data set and your annotations, however after 300 epochs the IoU and L just fluctuates and does not go down. IoU is also extremely low. I have attached the terminal output of the first 40 epochs
[training_log.txt](https://github.com/ShiqiYu/libfacedetection.train/files/6900365/training_log.txt)

**Comment by fengyuentau** [comment]
Since our network is light-weight, it is hard for the light-weight network to further optimize itself as the training keeps going, which means it may reach local optimum.

**Comment by nhatanhphung** [comment]
I was not trying to load the pretrained model to further optimize, the training started from scratch and the loss would stay very high, and IoU would fluctuate between 0.11 and 0.16 mostly

**Comment by fengyuentau** [comment]
> I was not trying to load the pretrained model to further optimize, the training started from scratch and the loss would stay very high, and IoU would fluctuate between 0.11 and 0.16 mostly

What I am trying to say is the model's complexity limits its learning ability. But, of course, you can always think of some optimization to help it learn better.

**Comment by nhatanhphung** [comment]
when I tried to train the model with the default settings with the widerface data, the result is very wrong, low IoU, high loss, testing gives 5-6 bounding boxes for 1 singular face image.

**Comment by fengyuentau** [comment]
Can you provide your full train log and the test image?

**Comment by nhatanhphung** [comment]
I attached my train log earlier, I stopped after 40 epoch as there was no change
> [training_log.txt](https://github.com/ShiqiYu/libfacedetection.train/files/6900365/training_log.txt)
The image I used was from google
![image](https://user-images.githubusercontent.com/20840293/129037108-f863010d-4afe-4dc9-8758-85d5b05ecd02.png)

**Comment by fengyuentau** [comment]
Thanks for your log and the test image.

1. Since you stopped after 40 epochs, the model is not well trained. Please follow default settings and have it trained 500 epochs.
2. This model by default priorbox setting can detect faces of from ~10x10 pix to ~300x300 pix, but the face in the image (840x840 pix) you provide is way larger than this. You can try shrinking the image to 1/2 or even 1/3 and then run the detection.
